devicetypes/heltun/heltun-hls01-switch.src/heltun-hls01-switch.groovy

```diff
@@ -0,0 +1,304 @@
+	return result
+}
+
+def updated() {
+	initialize()
+}
+
+def checkParam() {
+	boolean needConfig = false
+	parameterMap().each {
+		if (state."$it.name" == null || state."$it.name".state == "defNotConfigured") {
+			state."$it.name" = [value: it.default as Integer, state: "defNotConfigured"]
+			needConfig = true
+		}
+		if (settings."$it.name" != null && (state."$it.name".value != settings."$it.name" as Integer || state."$it.name".state == "notConfigured")) {
+			state."$it.name".value = settings."$it.name" as Integer
+			state."$it.name".state = "notConfigured"
+			needConfig = true
+		}
+	}
+	if ( needConfig ) {
+		configParam()
+	}
+}
+
+private configParam() {
+	def cmds = []
+	for (parameter in parameterMap()) {
+	if ( state."$parameter.name"?.value != null && state."$parameter.name"?.state in ["notConfigured", "defNotConfigured"] ) { 
+			cmds << zwave.configurationV2.configurationSet(scaledConfigurationValue: state."$parameter.name".value, parameterNumber: parameter.paramNum, size: parameter.size).format()
+			cmds << zwave.configurationV2.configurationGet(parameterNumber: parameter.paramNum).format()
+			break
+		}
+	}
+	if (cmds) {
+		runIn(5, "checkParam")
+		sendHubCommand(cmds,500)
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.configurationv2.ConfigurationReport cmd) {
+	def parameter = parameterMap().find( {it.paramNum == cmd.parameterNumber } ).name
+	if (state."$parameter".value == cmd.scaledConfigurationValue){
+		state."$parameter".state = "configured"
+	}
+	else {
+		state."$parameter".state = "error"
+	}
+	configParam()
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.versionv1.VersionReport cmd) {
+	sendEvent(name: "firmwareVersion", value: (cmd.applicationVersion + (cmd.applicationSubVersion/100)))
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd) {
+	def locaScale = getTemperatureScale() //HubScale
+    def externalTemp = 1
+	def map = [:]
+	if (externalTemp == cmd.sensorType) {
+		def deviceScale = (cmd.scale == 1) ? "F" : "C" //DeviceScale
+		def deviceTemp = cmd.scaledSensorValue
+		def scaledTemp = (deviceScale == locaScale) ? deviceTemp : (deviceScale == "F" ? roundC(fahrenheitToCelsius(deviceTemp)) : celsiusToFahrenheit(deviceTemp).toDouble().round(0).toInteger())
+		map.name = "temperature"
+		map.value = scaledTemp
+		map.unit = locaScale
+		sendEvent(map)
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.meterv3.MeterReport cmd) {
+	def map = [:]
+	if (cmd.meterType == 1) {
+		if (cmd.scale == 0) {
+			map.name = "energy"
+			map.value = cmd.scaledMeterValue
+			map.unit = "kWh"
+			sendEvent(map)
+		} else if (cmd.scale == 2) {
+			map.name = "power"
+			map.value = Math.round(cmd.scaledMeterValue)
+			map.unit = "W"
+			sendEvent(map)
+		}else if (cmd.scale == 4) {
+			map.name = "voltage"
+			map.value = Math.round(cmd.scaledMeterValue)
+			map.unit = "V"
+			sendEvent(map)
+		}else if (cmd.scale == 5) {
+			map.name = "current"
```

Comment:
if you want this value to be visible to users or usable in automations you will need to define a custom capability and a custom presentation

Reply:
No, I don't want to, its a new feature and older models doesn't have this feature